frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/details-card.tsx

```diff
@@ -1,22 +1,28 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
+import { Base64 } from 'js-base64';
 import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 import { SubscriptionModel } from '@console/operator-lifecycle-manager';
 import {
   withDashboardResources,
   DashboardItemProps,
 } from '@console/internal/components/dashboard/with-dashboard-resources';
 import { FirehoseResource, FirehoseResult } from '@console/internal/components/utils';
-import { getName, getInfrastructurePlatform } from '@console/shared';
+import { getName, getInfrastructurePlatform, useDeepCompareMemoize } from '@console/shared';
 import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
 import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
 import DetailItem from '@console/shared/src/components/dashboard/details-card/DetailItem';
 import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
 import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
 import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
-import { InfrastructureModel } from '@console/internal/models';
+import {
+  useK8sWatchResource,
+  WatchK8sResource,
+} from '@console/internal/components/utils/k8s-watch-hook';
```

Comment:
We are already passing `watchK8sResource` & `stopWatchK8sResource` to the card as a prop. We can use that directly. We don't have to import these hooks here, right ?
^^ @bipuladh

Reply:
agreed